jobmaster/dm/task_manager.go

```diff
@@ -73,10 +75,10 @@ func (tm *TaskManager) OperateTask(ctx context.Context, op OperateType, jobCfg *
 
 	var stage metadata.TaskStage
 	switch op {
+	case Create, Update:
+		return tm.jobStore.Put(ctx, metadata.NewJob(jobCfg))
 	case Delete:
 		return tm.jobStore.Delete(ctx)
-	case Update:
-		return tm.jobStore.Put(ctx, metadata.NewJob(jobCfg))
 	case Resume:
 		stage = metadata.StageRunning
 	case Pause:
```

Comment:
line 94: 
> UpdateTaskStatus is called when receive task status from worker

What does worker mean, JobMaster?

Reply:
It is in OnWorkerOnline, which we get the status of the task and then updateTaskStatus